mobile/src/main/java/ch/epfl/sdp/mobile/ui/play/PlayScreen.kt

```diff
@@ -24,8 +31,30 @@ import ch.epfl.sdp.mobile.ui.PawniesIcons
 fun PlayScreen(
     state: PlayScreenState,
     modifier: Modifier = Modifier,
-    contentPadding: PaddingValues = PaddingValues(),
+    contentPadding: PaddingValues = PaddingValues()
 ) {
+  val lazyColumnState = rememberLazyListState()
+  val strings = LocalLocalizedStrings.current
+  Column(modifier) {
+    Text(
+        text = strings.playOnlineGames,
+        color = MaterialTheme.colors.primary,
+        style = MaterialTheme.typography.h4,
+        modifier = Modifier.padding(horizontal = 16.dp, vertical = 32.dp))
+    LazyColumn(
+        state = lazyColumnState,
+        verticalArrangement = Arrangement.Top,
+        horizontalAlignment = Alignment.CenterHorizontally,
+        modifier = modifier,
```

Comment:
Bug : You must not use `modifier` if this is not the top-level composable.
```suggestion
        modifier = Modifier,
```